### ISSUE TITLE
Navigation buttons in every slide?

### ISSUE THREAD

**Issue by rhedman** [NickPiscitelli/Glider.js#200]
I'm wondering if it's possible to have a set of next/prev buttons in every slide instead av two global ones? Just adding them breaks the Glider and I've tried to define them as an array but doesn't work. The reason for this is that I want to use the Glider as a step-by-step mini course interface and the nav buttons need to be contextual.

Thank you!

**Comment by rhedman** [comment]
I've figured it out, here is my solution:

`
const nextBtns = document.querySelectorAll('.js-glider-next');
const prevBtns = document.querySelectorAll('.js-glider-prev');

let slideIndex = 0;

if (nextBtns) {
[].forEach.call(nextBtns, (nextBtn) => {
	nextBtn.addEventListener('click', () => {
		slideIndex += 1;
		GliderSingle.scrollItem(slideIndex, false);
	});
});
}

if (prevBtns) {
[].forEach.call(prevBtns, (prevBtn) => {
	prevBtn.addEventListener('click', () => {
		slideIndex -= 1;
		GliderSingle.scrollItem(slideIndex, false);
	});
});
}
`

**Comment by rhedman** [comment]
That would work too but it would require hidden paging buttons though.

**Comment by NickPiscitelli** [comment]
Yeah, but it will ensure all events fire as well as have better compatibility with any future releases.

Also, you can ask Glider for the current index with the `getCurrentSlide` method.

**Comment by rhedman** [comment]
OK hmm, that method is not mentioned under "Methods" in the documentation so I didn't know it existed.

**Comment by NickPiscitelli** [comment]
It was added after the initial release. Feel free to submit a PR adding the
update.

**Comment by rhedman** [comment]
So is this how you mean?
`GliderSingle.scrollItem(_.getCurrentSlide(), true);`
It seems to work

**Comment by rhedman** [comment]
Great, thank you :)
